squidDrone/tests/com/com_message_buffer_tests.cpp

```diff
@@ -0,0 +1,67 @@
+#include "gtest/gtest.h"
+#include "com_message_buffer.hpp"
+
+namespace{
+  // Derived class for testing queue behaviour
+  class ComMessageBufferTest: public com::ComMessageBuffer {
```

Comment:
Why do you have this class. You use it on some tests, but not all. 
It doesn't add anything for testing. It is a direct child

Reply:
You are right. I don´t need it anymore. Did some queue manipulation before, but switched now to testcases, that use the public member functions. 
Fixed with ...